### PR TITLE
Website v2

### PR DESCRIPTION
**Pull request by Nikos1508**
Use Astro instead of plain HTML-CSS. Create Eurohacks, About Us, and Home pages. Setting some of them as "coming soon" features and also working on other features and typos.

### CODE REVIEW

**Review by BudzioT — APPROVED** [review]

**Review by ThomasJPrice — APPROVED** [review]

**Review by GalaxyGamingBoy — APPROVED** [review]
LGTM!

**Review by gicorada — APPROVED** [review]

**Review by v1ctorio — CHANGES_REQUESTED** [review]

File: src/components/Footer.astro [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                            <td class="flex justify-end mr-3"> Italy: </td>
+                            <td> italy@europehacks.com </td>
+                        </tr>
+                        <tr>
+                            <td class="flex justify-end mr-3"> Germany: </td>
+                            <td> germany@europehacks.com </td>
+                        </tr>
+                        <tr>
+                            <td class="flex justify-end mr-3"> Greece: </td>
+                            <td> greece@europehacks.com </td>
```
[the remark this answers is not in the document]
Comment (Nikos1508) [reply src/components/Footer.astro]: I will change it asap

File: src/content/events/poland.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                "data": " Fill this "
+            }
+        },
+
+
+            
+        "en": {
+            "welcome": " Welcome to the Poland Hackathon page. Here you can find more information about the French Hackathon. ",
+            "organizers": "Organizing committee",
+            "translation": "For changin the language from English to Polish press this button:",
```
[the remark this answers is not in the document]
Comment (Nikos1508) [reply src/content/events/poland.json:52]: I will probably change that but it won't be shown anywhere so I don't think it is important for THIS update

File: src/components/Footer.astro [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                            <td class="flex justify-end mr-3"> Poland: </td>
+                            <td> poland@europehacks.org </td>
+                        </tr>
+                        <tr>
+                            <td class="flex justify-end mr-3"> Nikos(GR): </td>
+                            <td> nikos.tsilas@europehacks.org </td>
+                        </tr>
+                        <tr>
+                            <td class="flex justify-end mr-3"> Marios(GR): </td>
+                            <td> marios.mitsios@europehacks.org </td>
```
[the remark this answers is not in the document]
Comment (Nikos1508) [reply src/components/Footer.astro:57]: I can't understand what are you referring to here, sorry

File: src/content/events/germany.json [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        "contact": {
+            "email": "germany@eurohacks.org", 
+            "phone": "+49 15679411392 (not final)"
+        }
+    },
+    "lang": {
+        "origin": {
+            "welcome": "Wilkommen",
+            "organizers": "Organisatoren",
+            "translation": "For changin the language from German to English press this button:",
```
[the remark this answers is not in the document]
Comment (Nikos1508) [reply src/content/events/germany.json:18]: Same with the poland.json, I will change it but it's not that important for now so we could leave it as it is for now.

File: src/content/events/poland.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (v1ctorio) [reply src/content/events/poland.json:52]: Ok

File: src/components/Footer.astro (hunk 3 above)
[the remark this answers is not in the document]
Comment (v1ctorio) [reply src/components/Footer.astro:57]: That's not the domain ><

File: src/pages/index.astro [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+		/>
+
+		<FAQ
+			question=" Can I participate in the event if I am not near the venue (or not even the host city)?"
+			answer=" Whatever you want if it is about computer science. It could be something with Arduino, raspberry pi, or (something that you don’t have to get your own hardware) coding. This could be website developing, app developing, code an Arduino or making code which do something cool and something that YOU are proud of."
+		/>
+
+		<FAQ
+			question=" Do I have to get a laptop with me? "
+			answer=" Yes. It is a coding event, you have to get your devices (it could be a tablet or a laptop) so you can make your projects. We may be able to offer some laptop in 1 or 2 people but you have to get your own devices in the hackathon.  "
```
[the remark this answers is not in the document]
Comment (Nikos1508) [reply src/pages/index.astro:85]: Okay, yes, you are right. I made this FAQ without asking anyone. I will make a Google document so we can review FAQ questions before we launch our website. ❤️

File: src/pages/index.astro [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+		/>
+
+		<FAQ
+			question=" Do I have to speak English? "
+			answer=" No, you don’t have to speak English or any other languages except your native one but it would be really helpful for navigating yourself in the website and also communicate with other people from all over Europe during the hackathon. (It is an optional activity but deferentially a wonderful experience). "
+		/>
+
+		<FAQ
+			question=" How can I contact you if I have any question for the events? "
+			answer=" Don’t hesitate to send us an email with all your question. We will answer them as soon as possible (probably under 24h). You can either send an email in questions@eurohacks.org or in yours country email which is <country>@eurohacks.org. If we don’t answer after 3 days you can contact hackathon’s phone number (but please don’t use it for no important reason). "
```
[the remark this answers is not in the document]
Comment (Nikos1508) [reply src/pages/index.astro:70]: Okay, we will review it on Google Document

File: src/components/HackathonElement.astro [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+<div class="hackathon-card font-bold bg-marine-sapphire w-90vw my-4vw mx-5vw rounded-2xl justify-center">
+    <div class="flex flex-row justify-between">
+
+        <div class="flex flex-col">
+            <div class="flex flex-row m-3 text-soft-peach">
+                <img src={image} width="90px" height="64px">
+                <div class="text-5xl my-5 ml-4"> {countrytitle} </div>
+            </div>
+            <div class="text-3xl my-3 ml-4 text-soft-peach">
+                <br> <i class="fa-solid fa-envelope mr-2 my-3"></i> Mail: {country}@eurohakcs.org
```
[the remark this answers is not in the document]
Comment (Nikos1508) [reply src/components/HackathonElement.astro:14]: I will change it asap

File: src/components/Footer.astro (hunk 1 above)
[the remark this answers is not in the document]
Comment (Nikos1508) [reply src/components/Footer.astro]: Wait, I think that I have changed it. Press "edit file" and tell me if you can see the changes. idk why they are not shown here. I will try to fix it.

**Review by v1ctorio — APPROVED** [review]

**Comment by v1ctorio** [comment]
Ready to merge!

**Pull request merged**